server/settings/publish_plugins.py

```diff
@@ -49,6 +50,20 @@ class CollectFtrackCustomAttributeDataModel(BaseSettingsModel):
     )
 
 
+class CollectWebpublisherCredentialsModel(BaseSettingsModel):
```

Comment:
I would not put it under publishing plugin, but it's category `Webpublisher`...

Reply:
![image](https://github.com/ynput/ayon-ftrack/assets/43494761/a6f557e4-058e-4b2c-9c07-a057226020ad)